chart/config/conditions.yaml

```diff
@@ -0,0 +1,131 @@
+# - What happens if both database flags (embedded & external) are true,
+#   or both false ?
+#
+# - Why are we using two flags anyway ?
+
+# # ## ### ###### ######## #############
+
+instance_group_condition:
+  api:
+    cc_uploader: true
+    cloud_controller_ng: true
+    file_server: true
+    loggr-udp-forwarder: true
+    policy-server-internal: true
+    policy-server: true
+    route_registrar: true
+    statsd_injector: true
+  asactors:
+    operator:      features.autoscaler.enabled
+    scalingengine: features.autoscaler.enabled
+    scheduler:     features.autoscaler.enabled
+  asapi:
+    golangapiserver:  features.autoscaler.enabled
+    metricsforwarder: features.autoscaler.enabled
+    route_registrar:  features.autoscaler.enabled
+  asdatabase:
+    postgres: features.autoscaler.enabled && !features.autoscaler.mysql.enabled
+  asmetrics:
+    eventgenerator: features.autoscaler.enabled
+    metricsserver:  features.autoscaler.enabled
+  asnozzle:
+    metricsgateway: features.autoscaler.enabled
+  auctioneer:
+    auctioneer: !features.eirini.enabled
+  cc-worker:
+    cloud_controller_worker: true
+  credhub:
+    credhub:         features.credhub.enabled
+    route_registrar: features.credhub.enabled
+  diego-api:
+    bbs: true
+    locket: true
+    loggr-udp-forwarder: true
+    silk-controller: true
+  diego-cell:
+    bosh-dns-adapter:    !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    garden:              !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    loggr-udp-forwarder: !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    netmon:              !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    rep:                 !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    route_emitter:       !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    silk-daemon:         !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+    vxlan-policy-agent:  !features.eirini.enabled && !features.multiple_cluster_mode.control_plane.enabled
+  doppler:
+    doppler: true
+  log-api:
+    loggregator_trafficcontroller: true
+    reverse_log_proxy: true
+    reverse_log_proxy_gateway: true
+    route_registrar: true
+  log-cache:
+    log-cache-cf-auth-proxy: true
+    log-cache-gateway: true
+    log-cache: true
+    log-cache-nozzle: true
+    route_registrar: true
+  nats:
+    nats: true
+    nats-tls: true
+  router:
+    gorouter: true
+    loggr-udp-forwarder: true
+  routing-api:
+    routing-api: features.routing_api.enabled
+  scheduler:
+    cc_deployment_updater:        true
+    cloud_controller_clock:       true
+    loggr-syslog-binding-cache:   true
+    loggr-udp-forwarder:          true
+    service-discovery-controller: true
+    ssh_proxy:                    !features.eirini.enabled
```

Comment:
The value must be quoted here because `!foo` is YAMl syntax for a local tag called `foo`. The actual value of `ssh_proxy` in the current form is `null`.

Reply:
Will fix.

Reply:
Done with commit 7eb9b9ff3ea2e0cf8f54eab43f2d08fc5d0ee875